src/ca_biositing/pipeline/tests/USDA/test_usda_comprehensive.py

```
@@ -0,0 +1,533 @@
+        args.all = True
+
+    print("🚀 COMPREHENSIVE USDA ETL DIAGNOSTICS")
+    print("=" * 60)
+
+    try:
+        if args.api or args.all:
+            mapper = CommodityMapper()
+            mapped_commodities = mapper.get_mapped_commodity_ids()
+
+            tester = USDAAPITester()
+            api_results = tester.comprehensive_api_test(mapped_commodities)
```

Ruff (0.15.0)

[error] Local variable `api_results` is assigned to but never used

Remove assignment to unused variable `api_results`

(F841)